lib/build/web_components.dart

```diff
@@ -39,7 +39,7 @@ Asset generateWebComponentsBootstrap(Resolver resolver, Transform transform,
       document.head.querySelector('script[type="application/dart"]');
   for (var importPath in htmlImportRecorder.importPaths) {
     var import = new dom.Element.tag('link')
-      ..attributes = {'rel': 'import', 'href': importPath,};
+      ..attributes = (new Map<dynamic,String>()..addAll({'rel': 'import', 'href': importPath,}));
```

Comment:
That seems like a bug as well, according to the api docs `attributes` should be a `Map<String, String>`